source/write/bulk-write.txt

```diff
@@ -0,0 +1,370 @@
+it to ``bulk``:
+
+.. literalinclude:: /includes/write/bulk-write.cpp
+   :start-after: start-bulk-delete-many
+   :end-before: end-bulk-delete-many
+   :language: cpp
+   :dedent:
+
+Run the Bulk Operation
+----------------------
+
+To run a bulk operation, call the ``execute()`` method on an instance of the 
+``mongocxx::bulk_write`` class that contains your write models. By default, the
+``execute()`` method runs the operations in the order they're appended to the
+``mongocxx::bulk_write`` instance.
+
+The following example performs the :ref:`insert <cpp-bulk-insert-model>`,
+:ref:`update <cpp-bulk-update-model>`, :ref:`replace <cpp-bulk-replace-model>`,
+and :ref:`delete <cpp-bulk-delete-model>` operations specified in the preceding
+sections of this guide by appending each corresponding write model to an instance
+of ``mongocxx::bulk_write`` and calling the ``execute()`` method. Then, it prints
+the number of modified documents:
+
+.. io-code-block::
+
+   .. input:: /includes/write/bulk-write.cpp
+      :start-after: start-bulk-run
+      :end-before: end-bulk-run
+      :language: cpp
+      :dedent:
+      :emphasize-lines: 30
+
+   .. output::
+      
+      Modified documents: 2
+
+If any of the write operations fail, the {+driver-short+} raises a
+``mongocxx::bulk_write_exception`` and does not perform any further operations.
+
+.. tip::
+
+   To learn more about the ``modified_count()`` function, see the :ref:`cpp-bulk-return-value`
+   section of this guide.
+
+Customize Bulk Write Operations
+-------------------------------
+
+You can modify the behavior of the ``create_bulk_write()`` method by passing
+an instance of the ``mongocxx::options::bulk_write`` class as a parameter. The following
+table describes the fields you can set in a ``mongocxx::options::bulk_write``
+instance:
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Field
+     - Description
+
+   * - ``ordered``
+     - | If ``true``, the driver performs the write operations in the order
+         provided. If an error occurs, the remaining operations are not
+         attempted. 
+       |
+       | If ``false``, the driver performs the operations in an
+         arbitrary order and attempts to perform all operations.
+       | Defaults to ``true``.
+
+   * - ``bypass_document_validation``
+     - | Specifies whether the operation bypasses document-level validation. For more
+         information, see :manual:`Schema
+         Validation </core/schema-validation/#schema-validation>` in the MongoDB
+         Server manual.
+       | Defaults to ``false``.
+
+   * - ``write_concern``
+     - | Specifies the write concern for the bulk operation. For more information, see
+         :manual:`Write Concern </reference/write-concern/>` in the {+mdb-server+} manual.
+
+   * - ``comment``
+     - | A comment to attach to the operation. For more information, see the :manual:`delete command
```

Comment:
S: Mirror the other entries in the table by using a verb? Ex: Attaches a comment...